tests/mocks/repository/lib/mock_kvset.c

```diff
@@ -491,7 +491,7 @@ _kvset_cursor_next(struct element_source *es, void **element)
 static int valbuf[1 + CN_SMALL_VALUE_THRESHOLD];
 
 static merr_t
-_kvset_iter_next_val(
+_kvset_iter_val_get(
```

Comment:
Can we name this function get_val()? I think 3.0 should get rid of this reversed naming scheme. We can't even keep it consistent internally.